css/properties/scrollbar-gutter.json

```diff
@@ -22,8 +22,7 @@
             "opera": "mirror",
             "opera_android": "mirror",
             "safari": {
-              "version_added": false,
-              "impl_url": "https://webkit.org/b/167335"
+              "version_added": "17.1"
```

Comment:
```suggestion
              "version_added": "17.1",
              "flags": [
                {
                  "type": "preference",
                  "name": "#CSS-scrollerbar-gutter-property",
                  "value-to-set": true
                }
              ]
```

@queengooborg I'm not sure if what I wrote is correct, as I don't find any prompts for Safari in here: https://github.com/mdn/browser-compat-data/blob/235c0d7dcc6dd5f7d98470f28aa90cef2ce41115/schemas/compat-data-schema.md?plain=1#L418-L431

Like this: 
<img width="948" alt="image" src="https://github.com/mdn/browser-compat-data/assets/82551626/37621fad-e35a-4cb9-974f-dec4238c674b">

Reply:
Pretty close!  We don't have to format the name, so `CSS scrollerbar-gutter property` works for the flag name!

Reply:
Oh, I see.